fetcher/floatingip.go

```diff
@@ -7,8 +7,8 @@ import (
 var _ Fetcher = &floatingIP{}
 
 // NewFloatingIP creates a new fetcher that will collect pricing information on floating IPs.
-func NewFloatingIP(pricing *PriceProvider) Fetcher {
-	return &floatingIP{newBase(pricing, "floatingip", "location", "type")}
+func NewFloatingIP(pricing *PriceProvider, additionalLabels ...string) Fetcher {
+	return &floatingIP{newBase(pricing, "floatingip", []string{"location", "type"}, additionalLabels...)}
```

Comment:
In addition with the proposal from above, this could then become `newBase(pricing, "floatingip", "location", "type", additionalLabels...)`, which would look abit smoother, imo.